src/services/usfmImportService/storeChapter.ts

```
@@ -103,4 +155,36 @@ export const storeChapter = async (
       [],
     )
   }
+
+  // Generate embeddings in batch for all verses in the chapter
+  if (false) {
```

Biome (2.4.10)

[error] Unexpected constant condition.

(lint/correctness/noConstantCondition)